.eslintignore

```diff
@@ -3,7 +3,7 @@
 /build/
 /docs/
 /public/
-/packages/*/dist/**/*
```

Comment:
can drop this change i think

Reply:
Dropped the change. Should be synonymous: ignore the dir, or ignore everything in all the dirs in the dir 🙂